test-integration/rule/GraknTestServer.java

```diff
@@ -75,7 +75,13 @@
     private SessionStore sessionStore;
 
     public GraknTestServer() {
+        this("server/conf/grakn.properties", "test-integration/resources/cassandra-embedded.yaml");
```

Comment:
this is minor, but we should put this in a constant variable and refer to it, @lolski. makes it more obvious that it's default variable.